src/algos/time_iteration.jl

```diff
@@ -102,12 +102,17 @@ If the stochastic process for the model is not explicitly provided, the process
 # Returns
 * `dr`: Solved decision rule.
 """
-function time_iteration(model, dprocess::AbstractDiscretizedProcess, init_dr;
-      verbose::Bool=true, maxit::Int=100, tol::Float64=1e-8, details::Bool=true)
+function time_iteration(model::Model, dprocess::AbstractDiscretizedProcess,
+                        grid, init_dr;

```

Comment:
yes, it is intentional: the basic version of the function tries to take only "real" julia objects as arguments doesn't make any lookup in the symbolic data.